osv-scanner.toml

```diff
@@ -89,3 +89,9 @@ name = "stdlib"
 ecosystem = "Go"
 license.override = ["BSD-3-Clause"]
 reason = "Unidentified license, remove once https://github.com/google/deps.dev/issues/86 is resolved"
+
+[[PackageOverrides]]
+name = "sigs.k8s.io/gateway-api"
+version = "1.2.0-rc1"
+ecosystem = "Go"
+license.override = ["Apache-2.0"]
```

Comment:
can you add a reason?

Reply:
Not sure about the exact reason, but it seems that osv scanner has problem identifying the versions for rc releases.

https://github.com/envoyproxy/gateway/actions/runs/11065210699/job/30744231458?pr=4270

Reply:
can you address this? we can found out why in the future.